docs/src/main/paradox/mqtt-streaming.md

```diff
@@ -0,0 +1,87 @@
+# MQTT Streaming
+
+@@@ note { title="MQTT Streaming" }
+
+MQTT stands for MQ Telemetry Transport. It is a publish/subscribe messaging protocol, designed for constrained devices and low-bandwidth, high-latency or unreliable networks. The design principles are to minimize network bandwidth and device resource requirements whilst also attempting to ensure reliability and some degree of assurance of delivery. These principles also turn out to make the protocol ideal of the emerging “machine-to-machine” (M2M) or “Internet of Things” world of connected devices, and for mobile applications where bandwidth and battery power are at a premium.  
+
+Further information on [mqtt.org](https://mqtt.org/).
+
+@@@ 
+
+The Alpakka MQTT connector provides an Akka Stream flow to connect to MQTT brokers. In addition, a flow is provided so that you can implement your own MQTT server in the case where you do not wish to use a broker--MQTT is a fine protocol for directed client/server interactions, as well as having an intermediary broker.
+
+The library has no dependencies other than those of Akka Streams i.e. it is entirely reactive. As such, there should be a significant performance advantage given its pure-Akka foundations.
```

Comment:
Please add a reference to the Paho-based module, as well.
And add a reference to this module in `mqtt.md`...

Reply:
What would you like me to say in reference to Paho?

Reply:
"Alpakka contains another MQTT connector which is based on the Eclipse Paho client." (and in the other direction in `mqtt.md`)
Maybe point out some differences.